webapp/tests/karma/unit/controllers/reports.js

```diff
@@ -142,12 +142,13 @@ describe('ReportsCtrl controller', () => {
       scope.$broadcast('VerifyReport', true);
       return Promise.resolve().then(() => {
         chai.expect(post.callCount).to.equal(1);
-        chai.expect(post.args[0]).to.deep.equal([{
+        chai.expect(post.args[0][0]).to.deep.include({
```

Comment:
I'd think it would be better to use `sinon.useFakeTimers` and keep the `deepEqual` expectation, so we test we don't add additional properties.